src/payment/amazon-pay-setup.md

```diff
@@ -1,118 +1,128 @@
 ---
 title: Setting Up Amazon Pay
+redirect_from:
+  - /payment/amazon-pay-seller-central.html
 ---
 
-Amazon Pay and Login with Amazon work together to create a great buyer experience.
+Amazon Pay and Login with Amazon work together to create a great buyer experience. For more information, see the [Amazon Pay and Login with Amazon extension for Magento 2][1] documentation provided by Amazon.
 
-For more information, see the [Amazon Pay and Login with Amazon][1] documentation for Magento 2.
+## Prerequisites
+
+You must have the following resources available to complete the Amazon Pay setup in the Magento Admin:
+
+- A verified Amazon Pay merchant account ([https://pay.amazon.com/signup][4])
+- [Amazon Pay seller credentials][5]
 
 ## Connect to Amazon Pay
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. In the upper-right corner, set the **Scope** to establish where the Amazon Pay configuration applies.
+1. In the upper-left corner, set the **Scope** to establish where the Amazon Pay configuration applies.
 
 1. In the left panel, expand **Sales** and choose **Payment Methods**.
```

Comment:
```suggestion
1. In the left panel, expand ![Expansion selector]({% link images/images/btn-expand.png %}) **Sales** and choose **Payment Methods**.
```

Reply:
The left panel doesn't use that icon, it is only for page section expansion.